packages/ui/src/ui/components/ElementsTable/ElementsTable.scss

```diff
@@ -119,7 +119,6 @@ $thin-border: 1px solid var(--light-divider);
         padding: 0 8px;
 
         vertical-align: middle;
-        white-space: nowrap;
```

Comment:
ElementsTable is used in many places of the project. Are you sure it won't break any other places?

Reply:
double checked. I discovered that this change slightly affects the display of "Scheduling"

remade